### PR TITLE
Container test - run:servers

### PR DESCRIPTION
**Pull request by isacandrei**
I have sun three tests in a row by running the commands 

- run:servers
- run:tests
- docker rm -f client
- stop:servers

It seems like the error I had with the client container had something to do with my testing environment.
Please check if it works for you too.

### CODE REVIEW

File: RoboFile.php [hunk 1]
```diff
@@ -169,10 +169,10 @@ public function runTests($repoOwner, $repoName, $repoBranch)
 		$dockerTesting = new TestContainer;
 		$dockerTesting->set('name', 'client');
 		$dockerTesting->set('network', $dockerNetwork);
-		$dockerTesting->set('daemon', false);
+//		$dockerTesting->set('daemon', false);
 		$dockerTesting->set(
 			'ports', array(
-				'5900' => '5901'
+				'5901' => '5900'
```
Comment (jatitoam) [inline RoboFile.php]: I recall doing this change but still having the problem, so it works for you, right?

File: RoboFile.php [hunk 2]
```diff
@@ -169,10 +169,10 @@ public function runTests($repoOwner, $repoName, $repoBranch)
 		$dockerTesting = new TestContainer;
 		$dockerTesting->set('name', 'client');
 		$dockerTesting->set('network', $dockerNetwork);
-		$dockerTesting->set('daemon', false);
+//		$dockerTesting->set('daemon', false);
```
Comment (jatitoam) [inline RoboFile.php]: Can you please clean this up? Thanks

**Review by jatitoam — CHANGES_REQUESTED** [review]
Just a small comment and a question

**Comment by jatitoam** [comment]
check this out https://github.com/joomla-projects/joomla-testing/commit/cd820d132bc6e5c60ee1df372254b38ecca6456e

**Comment by jatitoam** [comment]
it seems this repo (my own) is just outdated

File: RoboFile.php (hunk 1 above)
Comment (isacandrei) [reply RoboFile.php]: Yes, it is mapped correctly now. I successfully connected to VNC.

**Comment by isacandrei** [comment]
I have done the cleanup. I will make the full request on the main repository also.

**Comment by jatitoam** [comment]
Actually that's my point, if you see the main repository it's updated like you just did (I forgot to make the change in my fork)

**Comment by jatitoam** [comment]
Heya @isacandrei  :)
It seems I missed this one but iirc it went directly to the upstream fork?

**Pull request closed without merging**